app/src/main/java/mobi/plantare/repository/ContributorsRepository.kt

```diff
@@ -0,0 +1,21 @@
+package mobi.plantare.repository
+
+import mobi.plantare.model.AppContributor
+import mobi.visamate.datasources.mock.ContributorsNetwork
+import retrofit2.Callback
+
+/**
+ * Created by gabriel on 22/2/18.
+ * Repository Layer for Contributors
+ * https://www.youtube.com/watch?v=FrteWKKVyzI&t=1910s
+ */
+class ContributorsRepository {
```

Comment:
Very important layer to have repository controls outside of activity very nice youtube link!!!!